### ISSUE TITLE
Potential keras import bug for YOLO v3

### ISSUE THREAD

**Issue by agibsonccc** [deeplearning4j/deeplearning4j#7255]
Import error on import, link provided internally for file for test suite (darknet conversion from keras)

org.deeplearning4j.exception.DL4JInvalidConfigException: Invalid configuration for layer (idx=-1, name=conv2d_46, type=ConvolutionLayer) for height dimension:  Invalid input configuration for kernel height. Require 0 < kH <= inHeight + 2*padH; got (kH=3, inHeight=2, padH=0)
Input type = InputTypeConvolutional(h=2,w=17,c=512), kernel = [3, 3], strides = [1, 1], padding = [0, 0], layer size (output channels) = 1024, convolution mo

Aha! Link: https://skymindai.aha.io/features/ND4J-55

**Comment by farizrahman4u** [comment]
@maxpumperla Do you have the model file for this?

**Comment by maxpumperla** [comment]
@farizrahman4u this one: https://github.com/qqwweee/keras-yolo3

**Comment by maxpumperla** [comment]
updated the title a bit... :P

**Comment by farizrahman4u** [comment]
The specific issue is fixed (will need samediff lambda layers to import the model)

**Comment by maxpumperla** [comment]
@farizrahman4u how was this fixed? this isn't very helpful in terms of documentation. what layer causes the issue? how can you be sure there are no remaining issues? did you implement the samediff layer as prototype or found another solution?

**Comment by farizrahman4u** [comment]
I will add the test resources for this and other PRs in a single pass later this week (i cant clone test resources from where I am at the moment). FWIW, its the inner `yolo_body` that should be imported, not the full model:

```
from yolo3.model import yolo_body
from keras.layers import *
import numpy as np


num_anchors = 9
num_classes = 20


def get_model():
    x = Input((100, 100, 3))
    model = yolo_body(x, num_anchors, num_classes)
    model.predict(np.zeros((32, 100, 100, 3)))
    return model

get_model().save('model.h5') # import this file

```
